src/public/modules/forms/base/components/verifiable-field.client.component.js

```diff
@@ -1,5 +1,10 @@
 'use strict'
-const { isEmpty, merge, keys } = require('lodash')
+const { isEmpty, merge, keys, get } = require('lodash')
+const {
+  resetVerifiedField,
+  triggerSendOtp,
+  verifyOtp,
+} = require('../../../../services/FieldVerificationService')
```

Comment:
this is a matter of personal preference, but I prefer the `import *` syntax for services

Reply:
This one I can do

Reply:
Fixed in 1f3bb05